src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,139 @@
+package calculator;
+
+import java.util.*;
+import java.lang.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid importing anything from the package java.lang](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4510206)